### ISSUE TITLE
Friends only as default

### ISSUE THREAD

**Issue by mithrand0** [ReactiveDrop/reactivedrop_public_src#259]
A lot of new users want to play with only their friends. They create a lobby for it. In the lobby screen, they forget to set the friends-only option. In game, they then start kick random pubs that join their game.

After asking, people suggest they don't read the permission option. They just start the game.

I would like to propose making the permission option default to 'friends-only', rather than 'public'. People who read the whole screen can easily change it to public.

**Comment by ywgATustcbbs** [comment]
Not only permission settings, but also many other ui buttons are unnoticeable, such as remove not button. Many new players know hoe to add bots but they don't know how to remove them. F9 is another example.

It is necessary to add a starter tutorial picture or something to show the button useage.

Is the game wants to have more players. It should treat new player as a fool. Add a ui tutorial like many online/mobile games do.

Btw, For me, i really can't relate main menu's "开始战役"(which is the direct translate For start campaign, yes i mean the first large text) to online game. I would definitely doubt where is online mode until i open that menu. So, why not change it to more obvious words "Play Online" witch tells everyone: "hey this is online?mode, click me!"

**Comment by pR0Ps** [comment]
My 2c:

I don't think changing the default is the right way of reducing the amount of in-game kicks. I suspect that on this screen in particular, people are just rushing to pick a map and start the game and don't read the entire thing properly. I also think that once the game starts that people aren't generally aware that they can change this setting. So they realize that they're hosting a public game once someone joins, but don't know how to (or that they even can) change the permissions.

Suggestion: have the public/private status displayed prominently on the ready screen right next to (or even as as part of) the "Invite" button.

That way it's right in a player's face when they go to invite friends to a game they think is private. There is also typically more downtime on the ready screen as players wait for others to pick characters/gear so players are more likely to look around while they're waiting and notice that the game is public/private. It would also mean that if they realize mid-game that the game is public they'll have an easy and obvious way of changing it for the next round.

**Comment by DukeDukeEm** [comment]
I'm for mithrand0's proposal.
Regarding ywgATustcbbs proposal:
* The F9 button isn't needed, there is a vote button at the bottom which changes this setting.
* Not being able to relate to a string means you should make a dedicated issue about it or directly communicate with the current chinese translator @zhongqingsong or even make this change in the open source itself, I'm pretty sure it's a win for everyone if better wordings are found.
* I would happily design a new UI, but I have no skills in creating it (and I also lack the time for learning it), and the ones knowing how to work with vgui2 are busy developing the rest of the game. So for now, it is what it is, a new UI is a big load of work.
Regarding pROPs input:
* changing the default sound well if the players are rushing through without reading
* putting big red signs up somewhere is dangerous, because out of another perspective there will be soon another thing which needs to be big and red, until everything is big and red

**Comment by BenLubar** [comment]
The vote button during briefing and F9 are the same screen.

**Comment by ywgATustcbbs** [comment]
> The vote button during briefing and F9 are the same screen.

Yes. But i see too many new players that do not know there is a vote system. When someone start a leader vote in a lobby, they just don't know how to vote. Or when someone is talking about ping, they keep asking how to check pings.

**Comment by BenLubar** [comment]
@DukeDukeEm I think vgui2 can do tooltips, but I'm not sure how to use them

**Comment by mithrand0** [comment]
>  ... and I also lack the time for learning it)

Redesigning a screen will also cost time re-implementing it in code, while changing a default value is a minute work. This is why I suggested only changing the default for now.

If this changing the default fixes only a small percentage of the kick non-friends issue, it should be done. We can always create a separate issue for a complete redesign, that we can brainstorm later. In the meantime, this the default could be a small step and quick win.
